rfc-013-code-repositories.md

```diff
@@ -0,0 +1,89 @@
+# Code repository standards
+
+## Summary
+
+A set of standards to be used for projects that will be shared within Made Tech or the greater technology community. These are projects hosted on GitHub, Bitbucket, Gitlab or similar public source control management.
+
+## Problem
+
+Made Tech code repositories (to be referred to as repos herein) do not have a common look and feel.
```

Comment:
Is that a problem?  I'm going to pop the why stack.  Why is not having a common look and feel bad?

Reply:
Common look and feel might be a bad choice of words. I think I was looking for consistency. If you don't have consistency, then people may include a `README.md,` but not the `LICENSE`. 

What do you think?

Reply:
Just trying to understand the underlying pain.  I'm guessing that it's something similar to this...

```
As an engineer who is maintaining a project, I want some set of documentation/features I can depend on being there so it is quicker for me to get started and I don't have to get answers from the original team.
```

But would be keen to hear what you view the problem to be.  That way there can be some confidence that the solution proposed in this RFC will solve your problem and isn't just process for the sake of process.

Reply:
I like your user story for the engineer, so I've used it as the problem description. This stop this  being thought of as an aesthetics issue.